tests/integration/test_with_portainer_service.py

```diff
@@ -2,13 +2,57 @@
 # pylint: disable=unused-argument
 
 import asyncio
+import os
+import subprocess
+import time
 from collections.abc import AsyncIterator
+from pathlib import Path
+from typing import Any, Callable
 
 import pytest
 from aiohttp import ClientSession
+from faker import Faker
 from yarl import URL
 
-from simcore_service_deployment_agent import portainer
+import docker
+from simcore_service_deployment_agent import exceptions, portainer
+
+pytest_plugins = [
+    "pytest_simcore.docker_registry",
+    "pytest_simcore.docker_swarm",
+    "pytest_simcore.schemas",
+    "pytest_simcore.repository_paths",
+    "fixtures.fixture_portainer",
+]
+
+
+@pytest.fixture(scope="session")
+def osparc_simcore_root_dir(request) -> Path:
+    return Path().cwd() / ".temp" / "osparc-simcore"
+
+
+def _run_cmd(cmd: str, **kwargs) -> str:
+    result = subprocess.run(
+        cmd, capture_output=True, check=True, shell=True, encoding="utf-8", **kwargs
+    )
+    assert result.returncode == 0
+    return result.stdout.rstrip() if result.stdout else ""
+
+
+@pytest.fixture
+async def portainer_baerer_code(
+    loop: asyncio.AbstractEventLoop,
```

Comment:
not sure you need to explicitely have that fixture here. and the correct name is
```event_loop``` see [https://pytest-asyncio.readthedocs.io/en/latest/concepts.html](https://pytest-asyncio.readthedocs.io/en/latest/concepts.html)

Reply:
true, thx, done